practices/managing-node-projects.md

```diff
@@ -10,7 +12,30 @@
 * [Dependency management tools](#dependency-management-tools)
 * [Handling dependencies from third parties](#handling-dependencies-from-third-parties)
 
-The following guide describes how we manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+The following guide describes how we use node and manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+
+
+## Specifying versions of node
+
+It is important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
+
+1. The [`engines`](https://docs.npmjs.com/files/package.json#engines) field in `package.json`:
+    * Important if authoring libraries, because `npm install` only _enforces_ compatibility on the basis of the `engines` field if the package is a dependency.
+    * Some deployment environments also respect it.
+    * It makes compatibility requirements explicit to developers working on your application.
+1. Using an `.nvmrc` file: An `.nvmrc` file is a configuration file for `nvm` (Node Version Manager). `nvm` enables developers to use different versions of node for different projects, on a per-directory basis. Projects should include an [`.nvmrc` file](https://github.com/creationix/nvm#nvmrc) in the root directory of the project to specify which version(s) of node are compatible. Developers can then run `nvm use` to use the right version of node, and [if using ZSH this shell script will run it automatically](https://github.com/creationix/nvm#calling-nvm-use-automatically-in-a-directory-with-a-nvmrc-file). Additionally [Travis respects `.nvmrc` files](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc), so using one will simplify your Travis configuration.
```

Comment:
"enables developers", "developers can run", "projects should include" - address all of those to "you", the reader